LambdaEngine/Source/Rendering/RenderGraphEditor.cpp

```diff
@@ -1748,13 +1768,8 @@ namespace LambdaEngine
 		static char renderStageNameBuffer[RENDER_STAGE_NAME_BUFFER_LENGTH];
 		static bool customRenderer = false;
 
-		static int32	selectedXOption	= 2;
-		static int32	selectedYOption	= 2;
-		static int32	selectedZOption	= 0;
-
-		static float32	xVariable = 1.0f;
-		static float32	yVariable = 1.0f;
-		static float32	zVariable = 1.0f;
+		static int32 selectedOptions[3]	= { 2, 2, 0 };			// Holds options for the x, y, and z dimensions.
+		static float variables[3]		= { 1.f, 1.f, 1.f };	// Holds the value of the x, y, and z dimension. 
```

Comment:
Naming is a bit confusing. Previous implementation used xyz in their names, maybe continue to do that to minimize confusion.
Instead of "selectedOptions" have "selectedXYZOptions" (or similar)
Instead of "variables" have "variablesXYZ" (or similar)